proto/search.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package chremoas.esi;
+
+service SearchQuery {
+    rpc GetSearch (SearchRequest) returns (SearchResponse) {
```

Comment:
Something says I would like `Search` more than GetSearch.  I see no reason to call out the http method that happens on the backend of this internal call.  I don't think I'd hate this TOO much though ;)

Reply:
Yeah, I'm down with that.